gnucash/import-export/csv-imp/assistant-csv-trans-import.cpp

```diff
@@ -1501,9 +1504,18 @@ make_column_header_model (bool multi_split)
         if (sanitize_trans_prop(col_type.first, multi_split) == col_type.first)
         {
             GtkTreeIter iter;
+            gchar *label;
             gtk_list_store_append (combostore, &iter);
+
+            if (formal && (col_type.first == GncTransPropType::DEPOSIT))
+                label = _("Debit");
+            else if (formal && (col_type.first == GncTransPropType::WITHDRAWAL))
+                label = _("Credit");
```

Comment:
I have dropped this.

Reply:
Let's not use "inverted". That usually means "divided into 1" e.g. 1/2 is the inverse of 2. I think accountants would use "reversed" for applying the amount with the sign changed.

That said, we absolutely *can* assume that the data follow Debit/Credit, Deposit/Withdrawal, Charge/Payment, or whatever convention because *that's what the user is telling us by selecting the column heading*. If there's a negative amount in the Credit column then the user should expect it to be the same as putting a negative amount in that column in the register. The tricky bit for infomal labels is getting the right set, which depends upon the account being imported into. Hardwiring Deposit/Withdrawal won't be right if the account isn't a bank account.
Now if the importer doesn't work like that but instead wants a signed amount that it puts in the debit or credit column depending on the sign then the heading should be "Amount" as you say.

Reply:
Resolving since the commit is dropped. @gjanssens should we change the Debit and Credit to Amount and Amount (Reversed) reflecting what the importer is really doing?

Reply:
Back to your previous comment first: the tricky bit of selecting the right informal label set. We simply can't because the csv importer is generic enough it can import transactions from and to multiple accounts. Importing into a single account is only a subset of that (which is hence also supported). In addition, assume we're importing into one single account, that account is not always known in the preview page, so we won't always be able to determine the proper set of informal labels in time to be useful.

So in a way Debit and Credit are better as they are always correct. But they also frequently lead to confusion as many users don't use these terms (and frankly I also continue to struggle with them as I don't use them enough). So I do vote for changing Deposit and Withdrawal to Amount and Amount (Reversed).